src/types/PlayerBlockAction.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/*
+ *
+ *  ____            _        _   __  __ _                  __  __ ____
+ * |  _ \ ___   ___| | _____| |_|  \/  (_)_ __   ___      |  \/  |  _ \
+ * | |_) / _ \ / __| |/ / _ \ __| |\/| | | '_ \ / _ \_____| |\/| | |_) |
+ * |  __/ (_) | (__|   <  __/ |_| |  | | | | | |  __/_____| |  | |  __/
+ * |_|   \___/ \___|_|\_\___|\__|_|  |_|_|_| |_|\___|     |_|  |_|_|
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * @author PocketMine Team
+ * @link http://www.pocketmine.net/
+ *
+ *
+*/
+
+namespace pocketmine\network\mcpe\protocol\types;
+
+use pocketmine\network\mcpe\protocol\serializer\PacketSerializer;
+
+/** This is used for PlayerAuthInput packet when the flags include PERFORM_BLOCK_ACTIONS */
+final class PlayerBlockAction{
+
+	public const START_BREAK = 0;
+	public const ABORT_BREAK = 1;
```

Comment:
Reminder for me to remove these constants and use the ones in PlayerAction when I wake up